cfn-resources/autogen/README.md

```diff
@@ -1,67 +1,68 @@
   ]
 }
 ```
 
 ### Step 2:
-Run below command to generate schema file that is required for ``cfn generagte`` in the next step.
+Run below command to generate schema file that is required for ``cfn generate`` in the next step.
 ```bash
     make schema
 ``` 
 
 this procedure should generate 2 files in the schema folder (custom db role for this example)
 
-![img.png](docs/img.png)
+![img.png](https://github.com/mongodb/terraform-provider-mongodbatlas/assets/122359335/c5d8f2b8-6e7c-4a28-b205-059e69327051)
 
-- mongodb-atlas-customdbrole.json : is the actual schema used to generate the resource
-- mongodb-atlas-customdbrole-req.json : contains metadata related to the code generation, like potentially required fields, and fields that might be needed by the Atlas go Client
+- mongodb-atlas-stream-connection.json : is the actual schema used to generate the resource
+- mongodb-atlas-stream-connection-req.json : contains metadata related to the code generation, like potentially required fields, and fields that might be needed by the Atlas go Client
 
 ### Step 3:
-Run make create command to generate Resource template which reads
-/configs/<resource>-schema.json and /configs/<resource>-req.json
+Run make create command to generate Resource template which reads the files generated in Step #2
+`/schemas/<resource>-schema.json` and `/schemas/<resource>-req.json`
 
 #### the make create requires 3 inputs:
 
-- schemaFileName = is the name of the resource in the file schema, check the file created in the /schemas folder, for this example the schemaFileName is "customdbrole" (mongodb-atlas-**customdbrole**.json)
+- schemaFileName = is the name of the resource in the file schema, check the file created in the /schemas folder, for this example the schemaFileName is "customdbrole" (mongodb-atlas-**streamconnection**.json)
 - dirName = is the name of the directory that will be used to store the resource, (dash case is the convention for the folder name) for this example we are using "custom-db-role"
-- typeName = is the type name, used by cloud formation, (MongoDB::Atlas::{TypeName}) MUST BE THE SAME AS DESPLAYED in the schema generated in step 2 check the property typeName in the mongodb-atlas-customdbrole.json, for this example we are usign **CustomDBRole**
+- typeName = is the partial resource type name, used by cloud formation, (MongoDB::Atlas::{TypeName}). This  MUST be the same as in the schema generated (mongodb-atlas-streamconnection.json) in step 2.
 
 ```bash
-        make create dirName=custom-db-role resourceName=customdbrole typeName=CustomDBRole
+        make create dirName=stream-connection resourceName=streamconnection typeName=StreamConnection
 ```
 
-After executing the command 2 inputs are requested, 
-- Language (go is required)
-- Go Import Path
 
+After executing the command you need to provide the Go import path, which will be prompted on the terminal, for example, `github.com/mongodb/mongodbatlas-cloudformation-resources/stream-connection`
```

Comment:
Could we also include the changes that we expect the developer still have to include in the generated code?

Reply:
not sure what you mean here? We mention in the limitataions that the generated code would need to be manually adjusted already, is that what you mean?

Reply:
I am wondering what you think about adding the limitations here in the readme (if they are not already there)

Reply:
I don't have a strong opinion but I don't see too much value in adding here as well as they are captured in our internal wiki already but feel free to disagree